inspirehep/dojson/experiments/rules.py

```diff
@@ -30,6 +30,7 @@
 from dojson.errors import IgnoreKey
 
 from inspirehep.utils.helpers import force_list
+from inspire_schemas.api import LiteratureBuilder
```

Comment:
Somehow I doubt that you're going to use the `LiteratureBuilder` while creating experiment records...